app/src/main/res/layout/detail_move_two.xml

```diff
@@ -0,0 +1,92 @@
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Would you consider using `ConstraintLayout` here?

Reply:
I know that we have to avoid deep view hierarchy in order to provide an efficient/fast layout. ConstraintLayout is a good option to keep the layout tree flattened.
I didn’t use it here, because I haven’t mastered it yet.
So, thats something I have to learn more.

Thanks for reviewing my code and for your time.

Reply:
Great!